dts/bindings/regulator/regulator-pmic.yaml

```diff
@@ -38,6 +39,10 @@ properties:
       to the modesel-reg.
 
 child-binding:
+  include:
+    - name: regulator.yaml
+      property-allowlist:
+        - regulator-boot-on
   description: Voltage output of PMIC controller regulator
```

Comment:
It isn't clear to me what this property is for? Is there more to this description we can add to clarify?

Reply:
The syntax here is including the `regulator.yaml` file into the binding definition, but selecting only the `regulator-boot-on` property from that binding. The property is documented there, but essentially will ensure the regulator is on at boot time